src/test/java/SetTest.java

```diff
@@ -0,0 +1,49 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author jeongheekim
+ * @date 3/6/24
+ */
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("numbers의 size는 3이다.")
+    void setSizeTest() {
+        assertEquals(numbers.size(), 3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    @DisplayName("원소들이 set에 존재한다.")
+    void elementsExistInSetTest(int input) {
+        assertThat(numbers.contains(input));
```

Comment:
input 에 포함되지 않는 4를 넣어도 테스트가 통과할것 같아요..!

assertThat 이후 검증 코드가 빠졌네요 🤔

assertj 의 api를 활용하는 방법도 있습니다!

```suggestion
        assertThat(numbers).contains(input);
```